liota/dcc_comms/mqtt_dcc_comms.py

```diff
@@ -74,18 +74,19 @@ def __init__(self, edge_system_name, url, port, identity=None, tls_conf=None, qo
 
         self.client_id = client_id
 
+        if self.client_id is None:
+            #  local_uuid generated will be the client ID
+            self.client_id = systemUUID().get_uuid(edge_system_name)
+            log.debug("Auto-Generated local uuid will be the client ID")
```

Comment:
Line 80 & 83: Should we also log the clientID if we're going to log at all?

Reply:
Fixed